Source/WebKit/WebProcess/Model/ModelProcessModelPlayer.cpp

```diff
@@ -75,6 +75,12 @@ void ModelProcessModelPlayer::didFinishLoading()
     m_client->didFinishLoading(*this);
 }
 
+void ModelProcessModelPlayer::DidUpdateEntityTransform(WebCore::TransformationMatrix transform)
+{
+    RELEASE_LOG(ModelElement, "%p - ModelProcessModelPlayer DidUpdateEntityTransform id=%" PRIu64, this, m_id.toUInt64());
```

Comment:
Does this really justify release logging?

Reply:
Removed